lib/matplotlib/tests/test_font_manager.py

```diff
@@ -87,3 +88,20 @@ def test_hinting_factor(factor):
     # Check that hinting only changes text layout by a small (10%) amount.
     np.testing.assert_allclose(hinted_font.get_width_height(), expected,
                                rtol=0.1)
+
+
+@pytest.mark.xfail(sys.platform != "win32",
```

Comment:
I think a skip would be better than an expected fail:
```suggestion
@pytest.mark.skipif(sys.platform != "win32",
```